augur/tasks/start_tasks.py

```diff
@@ -87,7 +87,7 @@ def primary_repo_collect_phase(repo_git):
 
     repo_task_group = group(
         repo_info_task,
-        chain(primary_repo_jobs | core_task_update_weight_util.s(repo_git=repo_git),secondary_repo_jobs,process_contributors.si()),
+        chain(primary_repo_jobs | issue_pr_task_update_weight_util.s(repo_git=repo_git),secondary_repo_jobs,process_contributors.si()),
         #facade_phase(logger,repo_git),
```

Comment:
Wouldn't the core data last collected still be NULL when the `issue_pr_task_update_weight_util` runs, since core collection isn't done yet. I'm confused as to why updating the secondary and facade weights are done in similar ways, but core is completely different. Could we just do it the same way for core?

Reply:
It is done in the same way for core. That chord is to update the issue and pr raw count from the primary tasks. The method at the end of the hook then updates it following the collection without updating the issue and pr raw count.

Reply:
core_task_success_util updates the core weight and secondary weight in the same way as secondary_task_success_util. The call to issue_pr_task_update_weight_util is just to update the raw count

Reply:
Oh, I see now. That makes sense